packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -723,7 +723,7 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 		return !this.disposed && this.socket.connected;
 	}
 
-	protected emitMessages(type: string, messages: IDocumentMessage[][]) {
+	protected emitMessages(type: string, messages: unknown[] | unknown[][]) {
 		// Only submit the op/signals if we are connected.
```

Comment:
This doesn't feel right.
`IDocumentMessage` is much nicer than unknown. Is there something else that should be used? Something for signals and leave IDocMsg for ops?